src/components/DeviceItem/DeviceItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { useMemo, useCallback } from "react";
+import get from "lodash/get";
+import { Title } from "@deskpro/app-sdk";
+import { Link, SimpleMDMLogo, Property, TwoProperties } from "../common";
```

Comment:
Use the PropertyRow in the App SDK wherever you use TwoProperties

Reply:
A couple of bugs should be fixed there. We created a story for it. I also added more comments there https://app.shortcut.com/deskpro/story/109670/app-sdk-fix-property-children.